site/en/docs/extensions/mv3/screen_capture/index.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Audio recording and screen capture"
+seoTitle: "Chrome Extensions: Audio recording and screen capture"
+date: 2023-04-14
+updated: 2023-04-14
+description: How to record audio or video from a tab, window or screen
+---
+
+APIs such as [chrome.tabCapture][tabcapture] and [getDisplayMedia][get-display-media] provide a
+variety of ways to monitor and record audio and video from a tab, window or screen.
+
+## Common Use Cases
+
+### Screen Recording {: #screen-recording}
+
+For screen recording, use the [getDisplayMedia][get-display-media] API. This provides the user with
+the ability to select which tab, window or screen they wish to share and provides a clear indication
+that recording is taking place.
+
+```js
+// User will be prompted to select what they would like to share.
+const stream = await navigator.mediaDevices.getDisplayMedia({ audio: true, video: true });
+```
+
+If called within a content script, recording will automatically end when the user navigates to a new
+page. To record in the background and across navigations, use an
+[offscreen document][offscreen-documents] with the <code>DISPLAY_MEDIA</code> reason.
+
+### Tab capture based on user gesture
+
+Some use cases need to capture the current tab following a user interaction
+with the extension's action icon. In these cases, it is not desirable to show a secondary dialog
```

Comment:
I don't understand the connection between clicking an extension icon and a secondary dialog being undesirable.